lib/get-changelog-using-wiki.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * A note script which extracts the changelog for the specified verion and repository
+ * this uses the Wiki yoast-cli php script. and expect that to be at /tmp/Wiki
+ *
+ * @param {Object} repositoryanme  object.
+ * @param {Object} verion     object.
+ * 
+ * @returns null
+ */
+const myArgs= process.argv
+var keyToSend = ""
+if (myArgs.length < 4) {
+	console.log("missinging arguments: <repository name>  <version>" )
+	return
+};
+
+const version = myArgs[3];
+
+switch (myArgs[2]) {
+	case 'wordpress-seo':
+		keyToSend = "A\r\n";
+		break;
+	case 'wordpress-seo-premium':
+		keyToSend = "B\r\n";
+		break;
+	case 'wpseo-news':
+		keyToSend = "C\r\n";
+		break;
+	case 'wpseo-video':
+		keyToSend = "D\r\n";
+		break;
+	case 'wpseo-news':
+		keyToSend = "E\r\n";
+		break;
+	case 'wordpress-seo-local':
+		keyToSend = "F\r\n";
+		break;
+	case 'yoast-acf-analysis':
+		keyToSend = "G\r\n";
+		break;
+	case 'yoastseo-amp':
+		keyToSend = "H\r\n";
+		break;
+	case 'javascript':
+		keyToSend = "I\r\n";
+		break;
+	case 'YoastSEO.js-premium-configuration':
+		keyToSend = "J\r\n";
+		break;
+	case 'yoast-test-helper':
+		keyToSend = "K\r\n";
+		break;
+	case 'duplicate-post':
+		keyToSend = "L\r\n";
+		break;
+	default:
+		console.log('Sorry, ' + myArgs[2] + ' is not something I know how to select from yoast-cli changelog:create.');
+		return
+};
+
+console.log (keyToSend);
+const { spawn } = require('child_process');
+const { exit } = require('process');
+const yoastcli = spawn('/tmp/Wiki/yoast-cli/yoast-cli.php', ['changelog:create'], {cwd:"/tmp/Wiki/yoast-cli/"});
+// set timeout to stop the script it input is not as expected for some reason.
+const timeout = setTimeout(function(){ yoastcli.kill()}, 30000);
+
+yoastcli.stdout.on('data', (data) => {
```

Comment:
Code here currently doesn't match the Yoast code style, we should plan in a short meeting to setup VSCode to automatically apply that codestyle.

Reply:
👍